autorel/dist_tarball_builder/docker_cli.py

```diff
@@ -0,0 +1,79 @@
+"""
+    Module : docker
+    This module provides abstraction layer over docker-py
+"""
+import logging
+import sys
+from docker import Client
+
+
+class Docker(object):
```

Comment:
So, the `instance` would be of `DistTarballBuilder` and methods would be `make` , `distcheck` etc. ?